dev/reference/performance/statement-summary.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Statement Summary Table
+summary: Learn about Statement Summary Table in TiDB
+category: reference
+---
+
+# Statement Summary Table
+
+To better handle SQL performance related issues, MySQL has provided [statement summary tables](https://dev.mysql.com/doc/refman/5.6/en/statement-summary-tables.html) in `performance_schema` to monitor SQL with statistics. Among these tables, `events_statements_summary_by_digest` is very useful in locating SQL problems with its abundant fields that include latency, execution times, rows scanned, full table scans, etc.
+
+TiDB provides the support for the `events_statements_summary_by_digest`
+table in v3.04. In this document, you will learn about this feature, and how to troubleshoot SQL performance issues with it.
```

Comment:
Starting from v3.0.4, TiDB provides the support for the `events_statements_summary_by_digest` table. In this document, you will learn about this feature, and how to troubleshoot SQL performance issues with it.